components/Form.js

```diff
@@ -0,0 +1,35 @@
+function handleSubmit(event) {
+  event.preventDefault();
+  console.log("submit clicked");
+}
+
+export default function Form({ handleAddNewWord }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    handleAddNewWord(
+      event.target.elements.word.value,
+      event.target.elements.translation.value
+    );
+    event.target.reset();
+    event.target.elements.word.focus();
```

Comment:
I think destructuring the `event.target.elements`could improve the readability.

```suggestion
    const { word, translation } = event.target.elements;

    handleAddNewWord(word.value, translation.value);
    event.target.reset();
    word.focus();
  }
```